src/luks2/reencrypt.rs

```diff
@@ -24,17 +26,23 @@ type ReencryptProgress = unsafe extern "C" fn(size: u64, offset: u64, *mut c_voi
 /// struct
 pub struct CryptParamsReencryptRef<'a> {
     #[allow(missing_docs)]
-    pub inner: libcryptsetup_rs_sys::crypt_params_reencrypt,
+    inner: libcryptsetup_rs_sys::crypt_params_reencrypt,
     #[allow(dead_code)]
     reference: &'a CryptParamsReencrypt,
     #[allow(dead_code)]
-    luks2_params: CryptParamsLuks2Ref<'a>,
+    luks2_params: Box<CryptParamsLuks2Ref<'a>>,
     #[allow(dead_code)]
     resilience_cstring: CString,
     #[allow(dead_code)]
     hash_cstring: CString,
 }
 
+impl<'a> CryptParamsReencryptRef<'a> {
```

Comment:
I'm not sure why it is necessary for return type to have ```mut``` instead of ```const``` modifier.

Reply:
You're right, this doesn't have to be that way. Resolved.